src/cargo/ops/cargo_output_metadata.rs

```diff
@@ -65,6 +72,8 @@ pub struct ExportInfo {
     version: u32,
     workspace_root: PathBuf,
     metadata: Option<toml::Value>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    workspace_profiles: Option<TomlProfiles>,
```

Comment:
Seems like this should be `profiles` instead of `workspace_profiles`

It looks like we only use the `workspace` prefix when qualifying things workspace root, workspace members) but otherwise we don't (e.g. metadata, resolve)